pkgs/build-support/kernel/make-initrd-ng-tool.nix

```diff
@@ -0,0 +1,10 @@
+{ rustPlatform }:
```

Comment:
You can drop the `cargoSha256` and use the `cargoLock` attribute instead:

```nix
buildRustPackage {
# ...
   cargoLock = {
     lockFile = ./Cargo.lock;
   };
# ...
}

```

This will also allow pulling in dependencies and updating them without having to care about the vendored package hashsum

Reply:
TIL, will do.